ufl/algorithms/apply_derivatives.py

```diff
@@ -1115,6 +1115,14 @@ def coordinate_derivative(self, o, f, dummy_w, dummy_v, dummy_cd):
                                                  rcache=self.rcaches[key]),
                                     o_[1], o_[2], o_[3])
 
+    def base_form_coordinate_derivative(self, o, f, dummy_w, dummy_v, dummy_cd):
```

Comment:
I don't think we should have a docstring here. The handlers of `DerivativeRuleDispatcher` do not have docstrings, probably because the class and handler names are sufficiently obvious for understanding what's going on. I am just being consistent with the existing code.